RestrictEvents/RestrictEvents.cpp

```diff
@@ -263,6 +260,54 @@ struct RestrictEventsPolicy {
 		return true;
 	}
 
+	static void getBlockedProcesses() {
+		// Updates procBlacklist with list of processes to block
+		char duip[128] { "auto" };
+		if (PE_parse_boot_argn("revblock", duip, sizeof(duip))) {
+			DBGLOG("rev", "read revblock from boot-args");
+		} else if (readNvramVariable(NVRAM_PREFIX(LILU_VENDOR_GUID, "revblock"), u"revblock", &EfiRuntimeServices::LiluVendorGuid, duip, sizeof(duip))) {
+			DBGLOG("rev", "read revblock from NVRAM");
+		}
+
+		char *value = reinterpret_cast<char *>(&duip[0]);
+		value[sizeof(duip) - 1] = '\0';
+		int i = 0;
+
+		// Disable notification prompts for mismatched memory configuration on MacPro7,1
+		if (strstr(value, "pcie", strlen("pcie")) || strstr(value, "auto", strlen("auto"))) {
```

Comment:
Same idea as below for MP71.

Reply:
For this I do believe it's reasonable, as it's SMBIOS specific